components/humantask/org.wso2.carbon.humantask/src/main/java/org/wso2/carbon/humantask/core/engine/runtime/xpath/JaxpFunctionResolver.java

```diff
@@ -253,23 +315,41 @@ public Object evaluate(List args) throws XPathFunctionException {
             }
             return createHttUser(username);
         }
+
+        private String getUserNameFromCtx() {
+            String username = "";
+            GenericHumanRoleDAO actualOwners = null;
+            actualOwners = evalCtx.getGenericHumanRole(GenericHumanRoleDAO.GenericHumanRoleType.ACTUAL_OWNER);
+            for (OrganizationalEntityDAO oe : actualOwners.getOrgEntities()) {
+                if (OrganizationalEntityDAO.OrganizationalEntityType.USER.equals(oe.getOrgEntityType())) {
+                    username = oe.getName();
+                    break;
+                }
+            }
+            return username;
+        }
     }
 
     /**
      * Returns the business administrators of the task. It MUST evaluate to an empty htt:organizationalEntity in case of
      * an error. If the task name is not present the current task MUST be considered.
+     * In : task name (optional)
+     * Out : business administrators (htt:organizationalEntity)
      */
     public class GetBusinessAdministrators implements XPathFunction {
+
         public Object evaluate(List args) throws XPathFunctionException {
```

Comment:
We are implementing XPathFunction interface here that has defined a generic list type. We don't have control over the XPathFunction. Thank you.